src/main/java/com/google/sps/servlets/BlobstoreUploadUrlServlet.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 Google LLC
```

Comment:
Are changes in this file relevant to the end-to-end testing?

Reply:
I created that file only for the testing, I needed the blobstore URL in my test and I didn't know how to get it from java.